src/Components/Pages/RentCalculator/RentCalculator.tsx

```diff
@@ -0,0 +1,152 @@
+    if (showRentInput) {
+      setShowRentInput(false);
+    }
+    const value = e.target.value;
+    setRentInput(value);
+  };
+
+  const handleSubmit = () => {
+    setShowRentInput(true);
+    gtmPush("gce_rent_calculator_submit");
+  };
+
+  return (
+    <div id="rent-calculator-page">
+      <Header
+        showProgressBar={false}
+        title="Calculate your rent increase"
+        subtitle="If you are covered by Good Cause legislation, you have a right to limited rent
+            increases. Use this calculator to determine the allowable rent increase for your apartment under Good Cause."
+      ></Header>
+      <div className="content-section">
+        <div className="content-section__content">
+          <div className="rent-calculator-callout-box">
+            <span className="callout-box__header">
+              Find out how much your landlord can increase your rent
+            </span>
+            <div className="rent-input-container">
+              <TextInput
+                labelText="Enter the total monthly rent for your entire apartment"
+                type="money"
+                id="rent-input"
+                value={rentInput}
+                onChange={handleInputChange}
+                onWheel={(e) => {
+                  // prevents scroll incrementing value
+                  e.currentTarget.blur();
+                  e.stopPropagation();
+                  setTimeout(() => e.currentTarget.focus(), 0);
+                }}
+              />
+              <Button
+                type="submit"
+                variant="primary"
+                size="small"
+                labelText="Calculate"
+                onClick={handleSubmit}
+              />
+            </div>
+            <div className="rent-increase-container">
+              <span className="rent-increase-header">
+                Allowable rent increase amount:
+              </span>
+              <br />
+              <span className="rent-increase-result">
+                {rentInput && showRentInput ? (
+                  <>
+                    <span className="rent-increase-formula">
+                      {`${formatMoney(Number(rentInput))} + ${increase_pct}% =`}{" "}
+                    </span>
+                    <span className="rent-increase-amount">
+                      {`${formatMoney(
+                        Number(rentInput) * (1 + increase_pct / 100)
+                      )} `}
+                    </span>
+                  </>
+                ) : (
+                  <>{`Your current monthly rent + ${increase_pct}%`}</>
+                )}
+              </span>
+            </div>
+          </div>
+          <p className="rent-increase-explanation">
+            If your rent was increased after April 20, 2024 beyond the allowable
+            rent increase amount calculated above, your rent increase could be
+            found unreasonable by housing court.
+            <br /> <br />
```

Comment:
We ran into a bug on firefox where these double `<br/>` don't show up, so we needed to switch to using margin on `p` tags (https://github.com/JustFixNYC/gce-screener/pull/124)